interfaces/apparmor/template.go

```diff
@@ -699,6 +699,9 @@ profile snap-update-ns.###SNAP_INSTANCE_NAME### (attach_disconnected) {
   /tmp/ r,
   /tmp/.snap/{,**} rw,
 
+  # Allow reading /proc/version. For release.go WSL detection.
+  /proc/version r,
```

Comment:
Please use `@{PROC}/version`, btw otherwise LGTM. Would be nice if this was moved up near the other @{PROC} accesses.

Reply:
Will do, thank you for the quick review!